fe/fe-core/src/main/java/org/apache/doris/cloud/transaction/CloudGlobalTransactionMgr.java

```diff
@@ -1213,6 +1293,26 @@ public boolean commitAndPublishTransaction(DatabaseIf db, long transactionId,
             }
             throw e;
         } finally {
+            stopWatch.stop();
+            String detailMsg = "";
+            if (commitCostTimeStatisticMap.containsKey(transactionId)) {
```

Comment:
and when remove `commitCostTimeStatisticMap`?

Reply:
in org.apache.doris.cloud.transaction.CloudGlobalTransactionMgr#afterCommitTransaction